src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -281,5 +290,45 @@ private int ParseAuditLevel()
             _logger.Log(message);
             return 1;
         }
+
+        private enum NuGetAuditMode { Unknown, Direct, Transitive }
+
+        private NuGetAuditMode ParseAuditMode()
+        {
+            string? auditMode = _restoreAuditProperties.AuditMode?.Trim();
+
+            if (auditMode == null)
+            {
+                return NuGetAuditMode.Unknown;
+            }
+            else if (string.Equals("direct", auditMode, StringComparison.OrdinalIgnoreCase))
+            {
+                return NuGetAuditMode.Direct;
+            }
+            else if (string.Equals("transitive", auditMode, StringComparison.OrdinalIgnoreCase))
+            {
+                return NuGetAuditMode.Transitive;
+            }
+
+            string messageText = string.Format(Strings.Error_InvalidNuGetAuditModeValue, auditMode, "direct, transitive");
```

Comment:
The message may not be well formed if the audit mode is set to `string.Empty`. May be consider empty string as `NuGetAuditMode.Unknown`.